package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "start": "NODE_ENV=production nodemon server/server.js",
     "build": "NODE_ENV=production webpack",
-    "dev": "NODE_ENV=development concurrently \"nodemon server/server.js\" \"webpack serve --open \""
+    "dev": "concurrently \"NODE_ENV=development nodemon server/server.js\" \"NODE_ENV=development webpack-dev-server --open\""
```

Comment:
Nice you made it work on Windows. As I said this is not a good idea in general, we should not merge this line. I am going to fix it the correct way in the branch we are working.